src/lib/transition_frontier_controller_tests/test_ledger_catchup.ml

```diff
@@ -31,6 +32,10 @@ end)
 
 let%test_module "Ledger catchup" =
   ( module struct
+    let run () =
```

Comment:
Likewise, for all the transition_frontier_components, can you name the create and run functions as create_<Transition_frontier_component> and run_<Transition_frontier_component>?